### ISSUE TITLE
Screen brightness not working flawless

### ISSUE THREAD

**Issue by sonst-was** [TransposonY/GestureSign#5]
Hi there,

just added a two gestures to change the screen brightness. The one for decreasing the brightness works without problems, but the one for increasing the brightness only goes up once or twice (not more than 30% of total brightness, not enough to be an actual change in brightness on my Surface Pro4). The gesture for increasing the brigjtness is recognized, but nothing happens after the first few times. If you need anymore information, please say would you need.

Greetings from Switzerland.

**Comment by TransposonY** [comment]
Your mean,for example,you would like to increase the brightness by 100%,actually system brightness only increased by 20%,even less than 20%?

**Comment by sonst-was** [comment]
No, I've set the brightness to increase in 10% steps (see image below), but the actual brightness does not increase above 20%. 

So for example I use one gesture to decrease the brightness to 0%. After that I'd like to increase it again to 100%. But regardless of how often I perform the "increase brightness"-gesture, the brightness will not get above 20% (the slider is stuck there).

![brightness](https://cloud.githubusercontent.com/assets/1457478/14231899/d480dca8-f994-11e5-82a1-630144bab29a.PNG)
![brightness2](https://cloud.githubusercontent.com/assets/1457478/14231898/d45e4aa8-f994-11e5-982b-db5ad3c695d8.PNG)

**Comment by TransposonY** [comment]
you can try v2.1

**Comment by sonst-was** [comment]
Just updated to v2.2. Now, its working a little bit better, but still not as expected.
First, decreasing the brightness by 10% if have to use the gesture exactly four times to go from maximum to minimum brightness.
Second, when increasing the brightness again I need to perform the gesture about ten times to get this far:
![brightness3](https://cloud.githubusercontent.com/assets/1457478/14359227/c6044282-fcf0-11e5-9f60-3701d97f259b.PNG)

After this, the brightness wont increase further.

**Comment by TransposonY** [comment]
Is it a common problem on Surface Pro 4?Maybe I need a Surface Pro 4 to test.

**Comment by sonst-was** [comment]
Not as far as I know.

Maybe you could ask around on reddit.com/r/surface. I'm pretty shure a lot people would help

**Comment by TransposonY** [comment]
Try disabling automatic brightness adjustments:
https://www.itsupportguides.com/windows-10/windows-10-how-to-disable-automatic-brightness-adjustments/

**Comment by sonst-was** [comment]
Just tried it but it didnt have any affect.

**Comment by lazer15** [comment]
In post 3 of this thread, how was a single touch added to that brightness gesture?  I can't seem to add a signal touch to custom gestures.  It seems to ignore single touches when you are in the process of making a custom gesture.  This is with version 3.2v.

**Comment by sonst-was** [comment]
I haven't tried it with v3.2 so far (I'm on 3.1.something).

Basically I just tap and hold the point with one finger and draw the curved line with another finger.

**Comment by JoseConseco** [comment]
I have same problem on Dell 15 7000, with different gestures setup.
- 3 fingers up - increase brightness 10%
- 3 downs - decrease brightness 10%
Decreasing brightness is actually to strong (it decreases brightness 20%), and increasing it 10% gives 5-8% increase. The gestures seems to be recognized well- one stroke = one brightness adjustment, but value is incorrect. Maybe the values are not set correctly? 
Maybe you do:
brightness*=110% 
instead of :
bright+=10%?
 Seems like this kind of problem.

**Comment by JoseConseco** [comment]
After restart everything seems to work ok now. Not sure why this happened and how it was fixed....
